src/java.base/unix/native/libnio/ch/Net.c

```diff
@@ -671,6 +671,16 @@ Java_sun_nio_ch_Net_joinOrDrop4(JNIEnv *env, jobject this, jboolean join, jobjec
         n = setsockopt(fdval(env,fdo), IPPROTO_IP, opt, optval, optlen);
     }
 #endif
+#ifdef _AIX
+    // workaround AIX bug where IP_ADD_MEMBERSHIP fails intermittently sometimes even more than one time...
```

Comment:
Adding a retry loop is okay, we have to retry on macOS too. Can you drop "even more than one time..." from the comment as it doesn't help.

Reply:
Of course! I dropped the end of the comment.